test/runnable/testxmm.d

```diff
@@ -2388,6 +2388,29 @@ void test23009()
     assert(R.array == correct);
 }
 
+/*****************************************/
+// https://issues.dlang.org/show_bug.cgi?id=23077
+
+float test23077(float x)
+{
+    short i = *cast(short*)&x;
+    ++i;
+    return *cast(float*)&i; // this cast is not allowed in @safe code
+}
+
+// https://issues.dlang.org/show_bug.cgi?id=23084
+
+short saturateSignedIntToSignedShort(int value) {
+    return cast(short) value;
+}
+
+__vector(int[4]) _mm_packs_epi32 (__vector(int[4]) a) {
```

Comment:
Rename this to avoid misleading correspondence with the signature of that Intel intrinsic.

Reply:
ok